src/language-html/embed.js

```diff
@@ -15,7 +15,7 @@ function embed(path, print, textToDoc, options) {
       // Inline JavaScript
       if (
         parent.type === "script" &&
-        ((!parent.attribs.lang && !parent.attribs.lang) ||
+        ((!parent.attribs.lang) ||
```

Comment:
(You have to run `yarn lint --fix` before `git commit`.)

Reply:
Sure !!